connector/sumconnector/go.mod

```
@@ -0,0 +1,82 @@
+module github.com/open-telemetry/opentelemetry-collector-contrib/connector/sumconnector
+
+go 1.21.0
+
+require (
+	github.com/open-telemetry/opentelemetry-collector-contrib/internal/filter v0.0.0-00010101000000-000000000000
```

```suggestion
	github.com/open-telemetry/opentelemetry-collector-contrib/internal/filter v0.103.0
```
The `replace` means we can just use the latest version here.